docs/content/documentation/content/shortcodes.md

```diff
@@ -72,6 +72,15 @@ There are two kinds of shortcodes:
 In both cases, the arguments must be named and they will all be passed to the template. 
 Parentheses are mandatory even if there are no arguments.
 
+Note that while a shortcode looks like a normal Tera template call, it is limited in several ways:
```

Comment:
It might be better to say that it is not actually Tera _at all_

Reply:
Sounds good -- I pushed some changes that clarify this a bit.